test/e2e/tests/simple-send.spec.js

```diff
@@ -23,8 +23,9 @@ describe('Simple send', function () {
           '0x985c30949c92df7a0bd42e0f3e3d539ece98db24',
         );
         await driver.fill('.unit-input__input', '1');
-        await driver.clickElement('[data-testid="page-container-footer-next"]');
-        await driver.clickElement('[data-testid="page-container-footer-next"]');
+        await driver.clickElement({ text: 'Next', tag: 'button' });
```

Comment:
Due to having to add a small delay after submitting a tx from send flow, the testid selector is *too fast* and results in trying to select the element that is hidden behind the loading indicator. This matches the other tests now and is *slow enough* to not break. Also all of this will be unnecessary once #10965 lands